### ISSUE TITLE
different Models and Flask Dev

### ISSUE THREAD

**Issue by fxx40a3** [fxx40a3/Research#16]
https://machinelearningprojects.net/milk-production-prediction/				
				
Machine learning projects at a glance				
	https://codegnan.com/machine-learning-projects/			
Artificial Intelligence Project For Students With Source Code and Example in 2024				
	https://www.fynd.academy/blog/artificial-intelligence-project-for-students			
				
				
Code	https://www.kaggle.com/code/eyabalaya/analysis-of-musical-trends-on-spotify			
Datasets	https://www.kaggle.com/datasets/leonardopena/top-spotify-songs-from-20102019-by-year			
				
				
Breast Cancer Detection App				
https://www.youtube.com/watch?v=SHgh2q0D3Vo&ab_channel=IndianAIProduction				
https://github.com/IndianAIProduction-Channel/Breast-Cancer-Detection-App				
https://indianaiproduction.com/breast-cancer-detection-using-machine-learning-classifier/				
				
https://github.com/vaibhavhariramani/Breast_Cancer_Detection_ML-with-Web-End-Deployment				
				
https://medium.com/geeky-bawa/end-to-end-deployment-of-breast-cancer-prediction-through-machine-learning-using-flask-5442ef4c586d				
https://vaibhavji.medium.com/end-to-end-deployment-of-breast-cancer-prediction-through-machine-learning-using-flask-part-2-67470b23dee9				
				
https://medium.com/swlh/deploying-breast-cancer-prediction-model-using-flask-apis-and-heroku-4fd3b65b2f26				
https://github.com/Muhd-Shahid/Breast-Cancer-Prediction-app-using-Flask-Python				
				
https://github.com/srimani-programmer/Breast-Cancer-Predictor/tree/master				
				
https://towardsdatascience.com/deploying-keras-models-using-tensorflow-serving-and-flask-508ba00f1037				
https://github.com/himanshurawlani/keras-and-tensorflow-serving				
				
Sentiment Analysis using Flask				
https://www.analyticsvidhya.com/blog/2024/03/how-to-deploy-a-machine-learning-model-using-flask/				
				
Heart-Disease-Diagnosis				
https://github.com/simplesaad/Heart-Disea				
				
https://blog.aiensured.com/part-iii-end-to-end-guide-for-heart-disease-prediction-deployment-with-flask/se-Diagnosis				
https://blog.aiensured.com/p/d71ac74d-ac67-4ef7-8527-48bceabc468e/				
				
https://medium.com/swlh/end-to-end-deployment-of-heart-disease-prediction-through-flask-with-machine-learning-algorithm-5cedd1e31b46				
				
https://github.com/swarnava-96/Heart-Disease-Prediction				
				
https://towardsdatascience.com/deploying-a-heart-failure-prediction-model-using-flask-and-heroku-55fdf51ee18e				
https://github.com/Ifeoluwa-hub/Heart-Failure-Prediction-and-Deployment-with-Flask-and-Heroku				
https://www.kaggle.com/datasets/andrewmvd/heart-failure-clinical-data				
				
https://www.sciencedirect.com/science/article/pii/S2666521223000145				
				
https://www.kaggle.com/datasets/andrewmvd/heart-failure-clinical-data/code				
				
https://vtupulse.com/				
				
				
Image Classification				
https://github.com/faizan170/tensorflow-image-classification-flask-deployment				
				
https://medium.com/@pg2196577/building-an-image-classifier-with-tensorflow-and-flask-a-beginners-journey-8f76ecd90603				
https://github.com/soumyapriyagoswami/Image-Classifier				
				
https://peerdh.com/uk/blogs/programming-insights/integrating-tensorflow-with-flask-for-real-time-image-classification				
				
https://buffml.com/multi-class-image-classification-flask-app-complete-project/				
https://drive.google.com/file/d/1GOsM_o01mvd8ljlwdcoiFPM6ZofA9FC-/view?usp=sharing%20noopener				
				
https://blog.gopenai.com/building-an-multiple-image-classification-web-app-with-cifar-10-and-flask-894ffcd9b121				
				
https://geekpython.in/flask-app-for-image-recognition				
https://github.com/Sachin-crypto/Flask_Image_Recognition				
https://www.geeksforgeeks.org/deploying-a-tensorflow-2-1-cnn-model-on-the-web-with-flask/				
				
face mask detection				
https://github.com/memudualimatou/FACE-MASK-DETECTION-WEB-APP-FLASK-PYTHON				
https://github.com/cabani/MaskedFace-Net				
				
https://github.com/GalileoParise/CV-Mask-detection				
				
https://www.geeksforgeeks.org/facemask-detection-using-tensorflow-in-python/				
				
https://github.com/pik1989/FaceMaskDetection/tree/main				
				
https://pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/				
				
https://makeoptim.com/en/deep-learning/yiai-object-detection				
				
Others				
https://www.geeksforgeeks.org/deploying-a-tensorflow-2-1-cnn-model-on-the-web-with-flask/				
https://medium.com/analytics-vidhya/deploy-ml-tensorflow-model-using-flask-backend-frontend-12a38ce53b7f				
https://github.com/muhammadarslanidrees/Deploying-Deep-Learning-Models-using-TensorFlow-Serving-with-Docker-and-Flask/tree/master				
				
https://www.analyticsvidhya.com/blog/2020/04/how-to-deploy-machine-learning-model-flask/				
				
Income Prediction App				
https://www.geeksforgeeks.org/deploy-machine-learning-model-using-flask/				
				
Question Classifier				
https://towardsdatascience.com/model-deployment-using-flask-c5dcbb6499c9
